### PR TITLE
refactor: remove lodash-es & replace node-sass to sass

### LINKED ISSUE #1655: Chore: remove lodash-es & replace node-sass to sass

### PR DESCRIPTION
**Pull request by zhangmo8**
## Checklist

- [ ] Fix linting errors
- [ ] Tests have been added / updated (or snapshots)

## Change information

remove lodash-es & replace node-sass to sas

## Issues

close #1655 
The issues you want to close, formatted as close #1.

## Related Links

Links related to this pr.

### CODE REVIEW

File: packages/varlet-cli/site/mobile/App.vue [hunk 1]
```diff
@@ -104,18 +104,20 @@ export default defineComponent({
     const showMenu: Ref<boolean> = ref(false)
     const showThemeMenu: Ref<boolean> = ref(false)
     const language: Ref<string> = ref('')
-    const languages: Ref<Record<string, string>> = ref(get(config, 'mobile.header.i18n'))
-    const themes: Ref<Record<string, any>[]> = ref(get(config, 'mobile.header.themes'))
+    const languages: Ref<Record<string, string>> = ref(config?.mobile.header.i18n ?? {})
```
Comment (chouchouji) [inline packages/varlet-cli/site/mobile/App.vue:107]: 这里是不是每个属性都加可选链操作符好些

File: packages/varlet-cli/src/node/config/vite.config.ts [hunk 2]
```diff
@@ -62,9 +59,9 @@ export function getDevConfig(varletConfig: Required<VarletConfig>): InlineConfig
     },
 
     server: {
-      port: get(varletConfig, 'port'),
+      port: varletConfig?.port,
       host: host === 'localhost' ? '0.0.0.0' : host,
-      proxy: get(varletConfig, 'proxy', {}),
```
Comment (chouchouji) [inline packages/varlet-cli/src/node/config/vite.config.ts:67]: lodash的处理是为`undefined`才返回默认值，我们是不是应该保持原来此逻辑
image

Comment (chouchouji) [reply packages/varlet-cli/src/node/config/vite.config.ts:67]: @haoziqaq 麻烦耗子哥也看下这里

Comment (zhangmo8) [reply packages/varlet-cli/src/node/config/vite.config.ts:67]: 不需要，没值有默认值在哪都有说法

**Review by chouchouji — APPROVED** [review]

**Comment by haoziqaq** [comment]
读取配置的部分尽量把可选链都加上。

**Review by haoziqaq — APPROVED** [review]

**Pull request merged**